downstairs/Cargo.toml

```diff
@@ -9,11 +9,17 @@ edition = "2018"
 anyhow = "1"
 bincode = "1.3"
 bytes = "1"
+chrono = { version = "0.4.19", features = [ "serde" ] }
 crucible = { path = "../upstairs" }
 crucible-common = { path = "../common" }
 crucible-protocol = { path = "../protocol" }
+dropshot = { git = "https://github.com/oxidecomputer/dropshot", branch = "main" }
 futures = "0.3"
 futures-core = "0.3"
+omicron-common = { git = "https://github.com/oxidecomputer/omicron", branch = "main" }
+oximeter-producer = { git = "https://github.com/oxidecomputer/omicron", branch = "main" }
+oximeter = { git = "https://github.com/oxidecomputer/omicron", branch = "main" }
+proc-macro2 = "1.0.32"
```

Comment:
Is this required? I don't see it used anywhere, so might be leftover from development.

Reply:
At some point during a bunch of trying to get public/private/usdt version change/etc work, I added it, but I just checked and a compile without it seems to work, so I'll pull it out.